kedro/framework/hooks/manager.py

```diff
@@ -10,14 +9,12 @@
 from .markers import HOOK_NAMESPACE
 from .specs import DataCatalogSpecs, DatasetSpecs, NodeSpecs, PipelineSpecs
 
-_hook_manager = None
-
 _PLUGIN_HOOKS = "kedro.hooks"  # entry-point to load hooks from for installed plugins
 
 logger = logging.getLogger(__name__)
 
 
-def _create_hook_manager() -> PluginManager:
+def create_hook_manager() -> PluginManager:
```

Comment:
Minor: Do we want to make this a public method? Just in case we want to remove it in future or something. In contrast, `_clear_hook_manager` has the initial underscore. Is it because it's effectively replacing the public `get_hook_manager`?

Reply:
Fair point. I initially kept it public to replace `get_hook_manager`, but maybe it's better to not have this as a public method at all. I'm trying to think of scenarios where a user might need to call `create_hook_manager` but can't really think of any atm..